src/docs/release/breaking-changes/target-platform-linux-windows.md

```diff
@@ -0,0 +1,153 @@
+  macOS,
+}
+```
+
+A `switch` statement only needed to handle these cases, and desktop applications which wanted to run on Linux or Windows usually had a shim something like this in their main:
+
+```dart
+// Sets a platform override for desktop to avoid exceptions. See
+// https://flutter.dev/desktop#target-platform-override for more info.
+void _enablePlatformOverrideForDesktop() {
+  if (!kIsWeb && (Platform.isWindows || Platform.isLinux)) {
+    debugDefaultTargetPlatformOverride = TargetPlatform.fuchsia;
+  }
+}
+
+void main() {
+  _enablePlatformOverrideForDesktop();
+  runApp(MyApp());
+}
+```
+
+## Description of change
+
+The [`TargetPlatform`][] enum is now defined as:
+
+```dart
+enum TargetPlatform {
+  android,
+  fuchsia,
+  iOS,
+  linux, // new value
+  macOS,
+  windows, // new value
+}
+```
+
+And the shim in `main()` is no longer required.
+
+This can cause the Dart analyzer to give the [`missing_enum_constant_in_switch`][]
+warning for switch statements which don't include a `default` case. Writing a
+switch without a `default:` case is the recommended way to handle enums, since
+the analyzer can then help you find any cases which aren't handled.
+
+## Migration guide
+
+In order to migrate to the new enum, and avoid the analyzer's [`missing_enum_constant_in_switch`][] error, which looks like:
+
+```
+warning: Missing case clause for 'linux'. (missing_enum_constant_in_switch at [package] path/to/file.dart:111)
+```
+
+or 
+
+```
+warning: Missing case clause for 'windows'. (missing_enum_constant_in_switch at [package] path/to/file.dart:111)
+```
+
+Modify your code as follows: 
+
+Code before migration:
+
+<!-- skip -->
+```dart
+void dance(TargetPlatform platform) {
+  switch (platform) {
+    case TargetPlatform.android:
+      // Do Android dance.
+      break;
+    case TargetPlatform.fuchsia:
+      // Do Fuchsia dance.
+      break;
+    case TargetPlatform.iOS:
+      // Do iOS dance.
+      break;
+    case TargetPlatform.macOS:
+      // Do macOS dance.
+      break;
+  }
+}
+```
+
+Code after migration:
+
+<!-- skip -->
+```dart
+void dance(TargetPlatform platform) {
+  switch (platform) {
+    case TargetPlatform.android:
+      // Do Android dance.
+      break;
+    case TargetPlatform.fuchsia:
+      // Do Fuchsia dance.
+      break;
+    case TargetPlatform.iOS:
+      // Do iOS dance.
+      break;
+    case TargetPlatform.linux: // new case
+      // Do Linux dance.
+      break;
+    case TargetPlatform.macOS:
+      // Do macOS dance.
+      break;
+    case TargetPlatform.windows: // new case
+      // Do Windows dance.
+      break;
+  }
+}
+```
+
+Having `default:` cases in such switch statements isn't recommended, because
+then the analyzer can't help you find all the cases which need to be handled.
+
+Also, any shims like the one above that set the
+[`debugDefaultTargetPlatformOverride`][] are no longer needed for Linux and Windows
+applications.
+
+## Timeline
+
+This change was made in March 2020.
```

Comment:
Do you have the release number? We can't merge this PR until we have an actual release #.

Reply:
No, not yet.  The PR is not yet committed.  Once it is, I'll have a release number.